### PR TITLE
[skip ci] README-MULTISITE: Fix syntax issues reported by markdownlint

### PR DESCRIPTION
**Pull request by BenoitKnecht**
Some parts of [README-MULTISITE.md](README-MULTISITE.md) didn't render properly, and [markdownlint](https://github.com/DavidAnson/markdownlint) (which is used by [GitHub's Super Linter](https://github.com/marketplace/actions/super-linter)) reported a bunch of syntax and style issues.

These commits fix those issues.

### CODE REVIEW

**Comment by guits** [comment]
@BenoitKnecht could you please squash your commits? Thanks!

**Comment by BenoitKnecht** [comment]
Done.

**Comment by dsavineau** [comment]
jenkins test docs

File: README-MULTISITE.md [hunk 1]
```diff
@@ -116,181 +119,184 @@ A secondary zone pulls a realm in order to sync data to it.
...
+    rgw_zone_user: edward.lewis
+    rgw_zone_user_display_name: "Edward Lewis"
 
-system_access_key: 6kWkikvapSnHyE22P7nO
-system_secret_key: MGecsMrWtKZgngOHZdrd6d3JxGO5CPWgT2lcnpSt
-```
-**Note:** `rgw_zonemaster` should have the value of `true` and `rgw_zonesecondary` should be `false`. Both values always need to be defined when running multisite.
+    system_access_key: 6kWkikvapSnHyE22P7nO
+    system_secret_key: MGecsMrWtKZgngOHZdrd6d3JxGO5CPWgT2lcnpSt
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:165]: Any reason for adding extra space characters there ?

File: README-MULTISITE.md [hunk 2]
```diff
@@ -116,181 +119,184 @@ A secondary zone pulls a realm in order to sync data to it.
...
+    system_secret_key: MGecsMrWtKZgngOHZdrd6d3JxGO5CPWgT2lcnpSt
 
-rgw_pull_proto: http
-rgw_pull_port: 8080
-rgw_pullhost: rgw-001-hostname
-```
+    rgw_pull_proto: http
+    rgw_pull_port: 8080
+    rgw_pullhost: rgw-001-hostname
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:203]: ditto

File: README-MULTISITE.md [hunk 3]
```diff
@@ -116,181 +119,184 @@ A secondary zone pulls a realm in order to sync data to it.
...
 
-```
-echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys-realm-1.txt
-echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys-realm-1.txt
+    ```bash
+    echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys-realm-1.txt
+    echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys-realm-1.txt
+
+    echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys-realm-2.txt
+    echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys-realm-2.txt
```
Comment (dsavineau) [inline README-MULTISITE.md:235]: ditto

File: README-MULTISITE.md [hunk 4]
```diff
@@ -116,181 +119,184 @@ A secondary zone pulls a realm in order to sync data to it.
...
-echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys-realm-2.txt
-```
 2. Edit `group_vars/all.yml` for the cluster
 
-```
-rgw_multisite: true
-```
+    ```yaml
+    rgw_multisite: true
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:242]: ditto

File: README-MULTISITE.md [hunk 5]
```diff
@@ -116,181 +119,184 @@ A secondary zone pulls a realm in order to sync data to it.
...
+    rgw_realm: france
+    rgw_zonegroup: idf
+    rgw_zone: paris
+    radosgw_address: "{{ _radosgw_address }}"
+    radosgw_frontend_port: 8080
+    rgw_zone_user: jacques.chirac
+    rgw_zone_user_display_name: "Jacques Chirac"
+    system_access_key: P9Eb6S8XNyo4dtZZUUMy
+    system_secret_key: qqHCUtfdNnpHq3PZRHW5un9l0bEBM812Uhow0XfB
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:264]: ditto

File: README-MULTISITE.md [hunk 6]
```diff
@@ -116,181 +119,184 @@ A secondary zone pulls a realm in order to sync data to it.
...
+    rgw_realm: usa
+    rgw_zonegroup: alaska
+    rgw_zone: juneau
+    radosgw_address: "{{ _radosgw_address }}"
+    radosgw_frontend_port: 8080
+    rgw_zone_user: edward.lewis
+    rgw_zone_user_display_name: "Edward Lewis"
+    system_access_key: yu17wkvAx3B8Wyn08XoF
+    system_secret_key: 5YZfaSUPqxSNIkZQQA3lBZ495hnIV6k2HAz710BY
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:282]: ditto

File: README-MULTISITE.md [hunk 7]
```diff
@@ -300,146 +306,147 @@ The host_vars for the rgws in the second cluster would have `rgw_zonesecondary`
...
+    echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys-realm-1.txt
+    echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys-realm-1.txt
 
-echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys-realm-2.txt
-echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys-realm-2.txt
+    echo system_access_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 20 | head -n 1) > multi-site-keys-realm-2.txt
+    echo system_secret_key: $(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 40 | head -n 1) >> multi-site-keys-realm-2.txt
+
+    ...
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:339]: ditto

File: README-MULTISITE.md [hunk 8]
```diff
@@ -300,146 +306,147 @@ The host_vars for the rgws in the second cluster would have `rgw_zonesecondary`
...
-...
-```
 2. Edit `group_vars/all.yml` for the cluster
 
-```
-rgw_multisite: true
-```
+    ```yaml
+    rgw_multisite: true
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:345]: ditto

File: README-MULTISITE.md [hunk 9]
```diff
@@ -300,146 +306,147 @@ The host_vars for the rgws in the second cluster would have `rgw_zonesecondary`
...
+    rgw_realm: france
+    rgw_zonegroup: idf
+    rgw_zone: paris
+    radosgw_address: "{{ _radosgw_address }}"
+    radosgw_frontend_port: 8080
+    rgw_zone_user: jacques.chirac
+    rgw_zone_user_display_name: "Jacques Chirac"
+    system_access_key: P9Eb6S8XNyo4dtZZUUMy
+    system_secret_key: qqHCUtfdNnpHq3PZRHW5un9l0bEBM812Uhow0XfB
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:367]: ditto

File: README-MULTISITE.md [hunk 10]
```diff
@@ -300,146 +306,147 @@ The host_vars for the rgws in the second cluster would have `rgw_zonesecondary`
...
+    rgw_realm: usa
+    rgw_zonegroup: alaska
+    rgw_zone: juneau
+    radosgw_address: "{{ _radosgw_address }}"
+    radosgw_frontend_port: 8080
+    rgw_zone_user: edward.lewis
+    rgw_zone_user_display_name: "Edward Lewis"
+    system_access_key: yu17wkvAx3B8Wyn08XoF
+    system_secret_key: 5YZfaSUPqxSNIkZQQA3lBZ495hnIV6k2HAz710BY
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:385]: ditto

File: README-MULTISITE.md [hunk 11]
```diff
@@ -300,146 +306,147 @@ The host_vars for the rgws in the second cluster would have `rgw_zonesecondary`
...
+    radosgw_address: "{{ _radosgw_address }}"
+    radosgw_frontend_port: 8080
+    rgw_zone_user: jacques.chirac
+    rgw_zone_user_display_name: "Jacques Chirac"
+    system_access_key: P9Eb6S8XNyo4dtZZUUMy
+    system_secret_key: qqHCUtfdNnpHq3PZRHW5un9l0bEBM812Uhow0XfB
+    rgw_pull_proto: http
+    rgw_pull_port: 8080
+    rgw_pullhost: rgw-001-hostname
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:410]: ditto

File: README-MULTISITE.md [hunk 12]
```diff
@@ -300,146 +306,147 @@ The host_vars for the rgws in the second cluster would have `rgw_zonesecondary`
...
+    radosgw_address: "{{ _radosgw_address }}"
+    radosgw_frontend_port: 8080
+    rgw_zone_user: edward.lewis
+    rgw_zone_user_display_name: "Edward Lewis"
+    system_access_key: yu17wkvAx3B8Wyn08XoF
+    system_secret_key: 5YZfaSUPqxSNIkZQQA3lBZ495hnIV6k2HAz710BY
+    rgw_pull_proto: http
+    rgw_pull_port: 8080
+    rgw_pullhost: rgw-002-hostname
+    ```
```
Comment (dsavineau) [inline README-MULTISITE.md:431]: ditto

File: README-MULTISITE.md (hunk 1 above)
Comment (BenoitKnecht) [reply README-MULTISITE.md:165]: Yes, that's for the ordered list items to be part of the same list.

For instance:

1. This is the first item:

```
Some code
```

2. This is not the second item, it's a new ordered list that starts at 2.

But this works:

1. This is the first item:

    ```
    Some code
    ```

2. This is the second item.

(You can inspect the HTML to see the difference.)

Comment (BenoitKnecht) [reply README-MULTISITE.md:165]: This was maybe clearer before I squashed the commits, this was originally added in f24e2078c77438c6d4b6c2bb1f7855b0eb71c1f7.

**Comment by guits** [comment]
jenkins test doc

**Comment by guits** [comment]
@BenoitKnecht

**Comment by BenoitKnecht** [comment]
@guits What's up?

**Comment by guits** [comment]
jenkins test docs

**Comment by guits** [comment]
@BenoitKnecht sorry, wrong ping 😅  . It was for @dsavineau  ...

**Comment by guits** [comment]
@dsavineau ?

**Review by guits — APPROVED** [review]

**Pull request merged**